frontend/scripts/react-components/dashboard-element/dashboard-panel/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
what does this do?

Reply:
It disables package-lock: https://docs.npmjs.com/misc/config#package-lock

Reply:
ok, but why is it in this particular folder? wouldn't it belong on the root of the project?

Reply:
oooh I missed that! yes it belongs to the root

Reply:
Does it need to be on the project's root? Or on the frontend's root? cc. @tiagojsag